functions/api/countLike.js

```diff
@@ -0,0 +1,44 @@
+import Sequelize from 'sequelize';
+import Like from './db/models/like';
+import db from './config/db';
+
+export async function main(event, context, callback) {
```

Comment:
I'm a bit torn on this.
We will never need just the like count. It always comes in the context of a comment. So I would suggest putting the like count in the get/listComment APIs, so that each comment that would be returned to the front end would be something like:

```
{
  content: '...',
  timestamp: 1234,
  createdAt: ...,
  user: {
    ...,
  },
  userLiked: boolean // only if logged in
}
```

Reply:
Also this way we prevent invoking lambda twice for each comment (once for the comment and user info, and another for the like count), which saves us money, and takes advantage of the speed of PostgreSQL.

Reply:
> I'm a bit torn on this.
> We will never need just the like count. It always comes in the context of a comment. So I would suggest putting the like count in the get/listComment APIs, so that each comment that would be returned to the front end would be something like:
> 
> ```
> {
>   content: '...',
>   timestamp: 1234,
>   createdAt: ...,
>   user: {
>     ...,
>   },
>   userLiked: boolean // only if logged in
> }
> ```


Very true. Now lambda needs to be invoked one extra time for each comment. I'll fix that!